statsmodels/graphics/tsaplots.py

```diff
@@ -7,8 +7,49 @@
 from statsmodels.graphics import utils
 from statsmodels.tsa.stattools import acf, pacf
 
+
+def _prepare_data_corr_plot(x, lags, zero):
+    irregular = False if zero else True
```

Comment:
`irregular = not zero`

however, we need `zero = bool(zero)` also below   if users use `zero=1`  (old style condition)